dhcpv6/option_clientlinklayeraddress.go

```diff
@@ -0,0 +1,46 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"github.com/insomniacslk/dhcp/iana"
```

Comment:
Better to keep it consistent with other files and have this import in the 2nd block
https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/dhcpv6.go#L8